package.json

```diff
@@ -102,7 +102,7 @@
     "argon2-browser": "^1.18.0",
     "arrive": "^2.4.1",
     "avsc": "^5.7.7",
-    "bcryptjs": "^2.4.3",
+    "bcrypt": "^5.1.1",
```

Comment:
`bcrypt`, which requires `node-gyp`, supports a far fewer subset of Node versions than `bcryptjs`.  Does the `bcryptjs` package also support the necessary functionality?